common/dub.sdl

```diff
@@ -4,5 +4,5 @@ preGenerateCommands "rdmd \"$PACKAGE_DIR/dubhash.d\""
 sourcePaths "src"
 importPaths "src"
 
-dependency "msgpack-d" version="~>1.0.0-beta.7"
+dependency "msgpack-d" version="~>1.0.1"

```

Comment:
as this is stable now (according to SemVer) we could use `~>1.0` which will be any version `>=1.0.0 <2.0.0` which should not get any backwards incompatible changes

Reply:
"Should" is the correct word here. This actually needs to point at 1.0.1 only. The CI fails for any version of msgpack-d that includes this commit: https://github.com/msgpack/msgpack-d/commit/a23181e14c0c4bc6a416ad0c712678c7ee8b3b21

Reply:
I'm not sure if this is a problem with msgpack-d, or a problem with the release build script. I'm not very familiar with cross-compiling Windows binaries with LDC.